modules/markup/html.go

```diff
@@ -92,6 +92,32 @@ func getIssueFullPattern() *regexp.Regexp {
 	return issueFullPattern
 }
 
+// CustomLinkURLSchemes allows for additional schemes to be detected when parsing links within text
+func CustomLinkURLSchemes(schemes []string) {
+	schemes = append(schemes, "http", "https")
```

Comment:
Do we need to add `mailto` here? It looks to currently render correctly.

Reply:
e-mail addresses are parsed separately. The `mailto:` URL scheme however must be added explicitly if wanted:

```
Hey, user anon@anon.com !                 <--- this doesn't need any settings
Hey, user mailto:anon@anon.com !          <--- this requires adding mailto to the list of custom schemes, and will need my PR to work
Hey, user [name](mailto:anon@anon.com)    <--- this too requires adding mailto to the list of custom schemes (not related to my PR)
```